src/main/java/com/bandwidth/iris/sdk/IrisConstants.java

```diff
@@ -1,28 +0,0 @@
-package com.bandwidth.iris.sdk;
-
-/**
- * Created by sbarstow on 11/11/14.
- */
-public class IrisConstants {
```

Comment:
I'm not a big fan of "Constants" classes. Too much of a catch-all and easily abused by stuffing all sorts in. 

All these constants are paths, right? How about calling it `IrisPath` (singular please). Or how about making this class a enum?